tests/integration/models/reddit/test_comment.py

```diff
@@ -218,6 +218,15 @@ def test_reply(self):
             assert not comment.is_root
             assert comment.parent_id == parent_comment.fullname
 
+    def test_reply__none(self):
+        comment = Comment(self.reddit, "d1616q2")
+        response_data = {"json": {"errors": [], "data": {"things": []}}}
+        with mock.patch.object(
```

Comment:
Is it difficult to use a cassette here and in the other test?

Reply:
Not here or the submission test, but AFAIK there’s no such thing as a quarantined message so it’s going to be tricky to get a cassette going for that particular test.

I’m not really a fan of cassettes anyway since they tend to add a lot of data that gets unused.

Reply:
If there isn't such a thing, does it make sense to even add a test for it then?

While you may not be a fan of cassettes I'd like to stick to that paradigm so that we can ensure the tests can be easily updatable through the removal of the existing cassettes.

Reply:
I suppose we can do away with the message test.

> While you may not be a fan of cassettes I'd like to stick to that paradigm so that we can ensure the tests can be easily updatable through the removal of the existing cassettes.

Tests written using cassettes aren’t any more easily “updatable” than a patching technique. If anything, I think using a cassette here makes managing the test harder because of the preparation work needed to produce the right cassette file that gets the test passing. Recreating cassettes can also be dangerous if the preparation work is not done right.

If someone wants to update a cassette they shouldn’t just blindly delete the cassette and rerun the test without first understanding exactly what the test does. I could add a cassette that references a quarantined submission, but if someone decides to regenerate the cassette without understanding the test logic either one of two things will happen to them: the cassette will be successfully created, the test passes, and the one who ran the test would have unknowingly replied to a submission in some quarantined subreddit; either that or the right cassette file will fail to be created because too much time has passed since the test was created and the submission has since “become a part of history”, as the reddit response would say.

With the mocks module we can have the ability to control both the response and the expectation in one place. With Betamax we can only control the expectation and extra work needs to be done to control the response. So using a cassette doesn’t automatically make things easier to manage and I think there’s nothing wrong with using a patch technique here.